python/ray/tests/test_basic.py

```diff
@@ -10,6 +10,8 @@
 import threading
 import time
 import pickle
+import uuid
+import tempfile
```

Comment:
I guess it's already not in alphabetical order, but we should keep it alphabetical.

Reply:
Agreed but the linter should do this. Want to make a PR that fixes it?